packages/ts-interface-generator/src/astGenerationHelper.ts

```diff
@@ -202,8 +202,8 @@ function generateSettingsInterface(
         propertySignature,
         buildJSDocStringFromLines(createJSDocCenterPart(association)),
       );
-      factory.createKeywordTypeNode(ts.SyntaxKind.StringKeyword),
-        interfaceProperties.push(propertySignature);
+      (factory.createKeywordTypeNode(ts.SyntaxKind.StringKeyword),
```

Comment:
my biggest doubts are about this one. The others work as intended despite the comma, but here line 205 does create a node which is never used